src/components/Posts/IndividualPost/IndividualPost.js

```diff
@@ -60,42 +57,57 @@ export const IndividualPost = () => {
       }
     };
     fetchPost();
-  }, []);
+    // eslint-disable-next-line
+  }, [post]);
```

Comment:
If use [post], it will infinitely call the function fetchPost, which means the content of post will automatically update without refreshing the page if owner update the post.

Reply:
Thank you for catching that!